pandas/io/parsers/c_parser_wrapper.py

```diff
@@ -379,40 +381,15 @@ def _concatenate_chunks(chunks: list[dict[int, ArrayLike]]) -> dict:
         arrs = [chunk.pop(name) for chunk in chunks]
         # Check each arr for consistent types.
         dtypes = {a.dtype for a in arrs}
-        # TODO: shouldn't we exclude all EA dtypes here?
         numpy_dtypes = {x for x in dtypes if not is_categorical_dtype(x)}
```

Comment:
Is it correct for `numpy_dtypes` to include the `ArrowDtype`s from the chunks?

Reply:
Doesn't really matter, we are only checking if we have more than one non categorical dtype. I'll rename the variable to avoid confusion